schemas/error-schema.json

```
@@ -66,6 +66,11 @@
               "enum": ["handled", "unhandled"],
               "readOnly": true
             },
+            "handling_stack": {
+              "type": "string",
+              "description": "Stacktrace of all instrumentation calls",
```

```suggestion
              "description": "Stacktrace of handling",
```
not sure what you mean by all instrumentation calls